images/expansionhunter_bw2/Dockerfile

```
@@ -0,0 +1,30 @@
+FROM ubuntu:20.04
+ARG DEBIAN_FRONTEND=noninteractive
+
+ARG VERSION=${VERSION:-5.0.0-bw2}
+
+# Install Git to clone the repository
+RUN apt-get update && \
+    apt-get install -y \
+    cmake \
+    g++ \
+    git \
+    libbz2-dev \
+    libcurl4-openssl-dev \
+    liblzma-dev \
+    libssl-dev \
+    zlib1g-dev
```

It's a good idea to clean out the apt caches after all this installation, as after here they're just extraneous bloat in the docker image:

```suggestion
    zlib1g-dev \
    && \
    apt-get clean -y && \
    rm -rf /var/lib/apt/lists/*
```